vernac/obligations.ml

```diff
@@ -487,13 +487,13 @@ let rec solve_obligation prg num tac =
         ++ str (string_of_list ", " (fun x -> string_of_int (succ x)) remaining));
   in
   let obl = subst_deps_obl obls obl in
-  let scope, kind = kind_of_obligation (snd obl.obl_status) in
+  let kind = kind_of_obligation (snd obl.obl_status) in
   let evd = Evd.from_ctx prg.prg_ctx in
   let evd = Evd.update_sigma_env evd (Global.env ()) in
   let auto n oblset tac = auto_solve_obligations n ~oblset tac in
   let proof_ending = Lemmas.Proof_ending.End_obligation (DeclareObl.{name = prg.prg_name; num; auto}) in
   let hook = DeclareDef.Hook.make (obligation_hook prg obl num auto) in
-  let info = Lemmas.Info.make ~hook ~proof_ending ~scope ~kind () in
+  let info = Lemmas.Info.make ~hook ~proof_ending ~kind () in
```

Comment:
Why change this?

Reply:
It is constant and coincides with the default value.

Reply:
Isn't it the non-default value?

Reply:
Oh indeed, let me fix... My eyes got confused by the long prefix.